build.sbt

```diff
@@ -43,12 +46,11 @@ lazy val core = project
 
 lazy val cassandraLauncher = project
   .in(file("cassandra-launcher"))
-  .enablePlugins(Common)
+  .enablePlugins(Common, ReproducibleBuildsPlugin)
   .disablePlugins(MimaPlugin)
   .settings(
     name := "pekko-persistence-cassandra-launcher",
-    Compile / managedResourceDirectories += (cassandraBundle / target).value,
-    Compile / managedResources += (cassandraBundle / Compile / packageBin).value)
+    Compile / unmanagedResources += (cassandraBundle / Compile / packageBin).value)
```

Comment:
This is PR change, can you take a look? thanks @raboof

Reply:
This does make sense as as those build results are 'unmanaged' from the perspective of the `cassandra-launcher` build.

I might worry sbt wouldn't correctly pick up that changes to the bundle should trigger a rebuild of the launcher, but perhaps that's sufficiently guaranteed since `cassandraBundle / Compile / packageBin` is still part of the graph.

In any case I don't think we expect many changes here, and CI shows it works well also when building 'from scratch'. I've also tried various experiments in sbt and didn't see any new/seriously problematic behavior.

For reference: I did notice that `sbt clean` does not clear `cassandra-bundle/target`, and it does not get rebuilt when it exists and you're building `sbt cassandraLauncher/package`, even if you made changes to the definition of the bundler. That was all already the case on the main branch, though, so not a regression.